sewer/dns_providers/cloudflare.py

```diff
@@ -149,3 +161,10 @@ def json(self):
                 )
 
         self.logger.info("delete_dns_record_success")
+
+    def _get_auth_header(self):
+        if self.CLOUDFLARE_TOKEN:
+            print(self.CLOUDFLARE_TOKEN)
```

Comment:
@moritz89 print forgotten after debugging?

Reply:
Haha, thanks for catching that